JetNews/app/src/main/java/com/example/jetnews/ui/home/HomeScreen.kt

```diff
@@ -253,15 +253,14 @@ private fun HomeScreenErrorAndContent(
 ) {
     if (posts.data != null) {
         PostList(posts.data, navigateTo, favorites, onToggleFavorite, modifier)
-    } else {
-        Box(Modifier.fillMaxSize()) {
-            if (!posts.hasError) {
-                // if there are no posts, and no error, let the user refresh manually
-                TextButton(onClick = onRefresh, modifier.fillMaxSize()) {
-                    Text("Tap to load content", textAlign = TextAlign.Center)
-                }
-            }
+    } else if (!posts.hasError) {
+        // if there are no posts, and no error, let the user refresh manually
+        TextButton(onClick = onRefresh, Modifier.fillMaxSize()) {
```

Comment:
Nit: what about using `modifier` here?

```suggestion
        TextButton(onClick = onRefresh, modifier.fillMaxSize()) {
```